### DISCUSSION TITLE
Can we unbundle the specification to allow independent evolution?

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/protocol-specifications#207] [Core Specification]
Beckn protocol specification consists of an Design Principles, API specificaion, Schema, Digital Signatures, Communication Protocol, Reference Architecture and a Governance Model. 

Given these are independently evolving components, it doesn't seem right to put them under one umbrella i.e Core.

Registry is already decoupled from core but it still contains schema, API and the signature specification in it. 

I have represented the various components of the beckn protocol specification in the below diagram

![Components](https://user-images.githubusercontent.com/52468749/150633883-ec47169a-df75-4304-aa08-61a7be4e38c0.png)

Can we move each components into separate repositories and track them independently?

@pramodkvarma @sjthnrk @venkatramanm @pkapustin

**Comment by pkapustin** [comment]
@core-wg-admin I think it is a good idea to unbundle these.

Also I think we need to explicitly mention:
- Adaptation for various transport protocols 
- Domain-specific specs
- Network-specific specs (policies)
